docs/audio-playback.md

```
@@ -0,0 +1,225 @@
+
+using MP3Speaker speaker = new();
+await speaker.PlayAsync(audioStream);
+```
+
+`PlayAsync` は再生が完了するまで待機します。
+
+> [!NOTE]
+> 1 つの `MP3Speaker` インスタンスで同時に複数の音声は再生できません。
+> 再生中に `PlayAsync` を呼び出すと `InvalidOperationException` が発生します。
+> 連続して読み上げる場合は前の `PlayAsync` の完了を待つか、インスタンスを分けてください。
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)